load/options.go

```diff
@@ -55,6 +58,9 @@ type Options struct {
 	//
 	// Default: ModeDefault.
 	Mode LoadMode
+	// Validator enables the immutability validation of the loaded config, and
+	// specifies its interval and error handler.
+	Validator *Validator

```

Comment:
- Should define option at loader repo https://github.com/tableauio/loader, but not here.
- Naming: `Validator` -> `ImmutableChecker`